k-distribution/include/ocaml/lexer.mll

```diff
@@ -23,6 +23,27 @@ rule token = parse
   | eof { EOF }
 
 { 
+
```

Comment:
this code is designed to speed up parsing of binary terms in ocaml by making lookups in the intern list happen in constant time.